node_modules/oae-admin/tenants/tenants.html

```diff
@@ -54,7 +54,7 @@
                                <div class="jeditable-field" data-alias="${tenant.alias|encodeForHTMLAttribute}" data-field="host">${tenant.host|encodeForHTML}</div>
                             </td>
                         {else}
-                            <td><a href="//${tenant.host}" title="${tenant.displayName|encodeForHTMLAttribute}" target="_blank">${tenant.host|encodeForHTML}</a></td>
```

Comment:
This title attribute isn't entirely redundant as it provides a readable tooltip for the link